libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -603,9 +608,11 @@ namespace SohImGui {
                 }
 
                 EnhancementCheckbox("Show Inputs", "gInputEnabled");
+                Tooltip("Shows currently pressed inputs on the bottom right of the screen");
                 EnhancementCheckbox("Rumble Enabled", "gRumbleEnabled");
 
                 EnhancementSliderFloat("Input Scale: %.1f", "##Input", "gInputScale", 1.0f, 3.0f, "", 1.0f, false);
+                Tooltip("Sets the scale of the displayed inputs from Show Inputs");
```

Comment:
If you're taking suggestions, "Sets the onscreen size" reads better to me, if I were somebody who didn't know what any of these settings mean.